crates/oxc_ast/src/serialize.rs

```diff
@@ -951,6 +951,21 @@ impl ESTree for JSXOpeningFragmentConverter<'_> {
 // TS
 // --------------------
 
+/// Serializer for `computed` field of `TSEnumMember`.
+#[ast_meta]
+#[estree(ts_type = "boolean", raw_deser = "false")]
```

Comment:
@overlookmotel Would you please help me to implement `raw_deser` here? I'm not sure how to touch `enum` variants in JS.

Reply:
Sure. I'll push a commit to this PR.

Reply:
Done. It's a dreadful hack!

Reply:
Thank you so much! 🙌🏻